<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThongTinNguoiDung</class>
 <widget class="QWidget" name="ThongTinNguoiDung">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ThongTinNguoiDung{
border-image: url(:/newPrefix/Images/thongtinnguoidung.png);
}
QLineEdit {
border:none;
    background-color: rgba(0, 0, 0, 0);
    border-bottom: 2px solid rgb(255, 184, 108);
	
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}
QComboBox {
border:none;
    background-color: rgba(0, 0, 0, 0);
    border-bottom: 2px solid rgb(255, 184, 108);
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}
QSpinBox {
border:none;
    background-color: rgba(0, 0, 0, 0);
    border-bottom: 2px solid rgb(255, 184, 108);
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}
QDoubleSpinBox {
border:none;
    background-color: rgba(0, 0, 0, 0);
    border-bottom: 2px solid rgb(255, 184, 108);
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}
QLabel{
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton{
background-color: rgba(0,0,0,0);
border-radius: 15px;
}
QPushButton:hover{
background-color: rgba(0,0,0,20);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tên</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTen">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Giới tính</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tuổi</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Chiều cao</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cân nặng</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxTuoi">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>18</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxGioiTinh">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Nam</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nữ</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spinBoxCanNang">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>440</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>30</number>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>70</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>390</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>(cm)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>440</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>(kg)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxChieuCao">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>400</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>250</number>
   </property>
   <property name="value">
    <number>170</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNext">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>580</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Tổng lượng calo tiêu thụ: </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTongLuongCalo">
   <property name="geometry">
    <rect>
     <x>274</x>
     <y>500</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
